src/xGadgetron/pGadgetron/tests/test1.txt

```diff
@@ -1,9 +1,9 @@
-2.212010e+02
-2.090210e+02
-2.090212e+02
-2.090210e+02
-2.155659e-07
-9.322899e-08
-1.430538e-06
-0.000000e+00
-0.000000e+00
+2.212586e+02
```

Comment:
are most of these actually zero? Do they need to change?

Reply:
?1: 5 values are close to zero (less than 1e-4)
?2: no need to change if equal within default relative tolerance of 2e-3